shared/src/components/question/answer.jsx

```diff
@@ -194,73 +136,26 @@ class Answer extends React.Component {
 
-        let body = '';
-        if (type === 'teacher-review') {
-            body = (
-                <div className="review-wrapper">
-                    <div className={cn('review-count', { 'green': isCorrect, 'red': !isCorrect })}>
-                        {selectedCount}
-                        <span className={cn('letter', { 'green': isCorrect, 'red': !isCorrect })}>
-                            {ALPHABET[iter]}
-                        </span>
-                    </div>
-                    <div className="answer-answer">
-                        <ArbitraryHtmlAndMath
-                            {...htmlAndMathProps}
-                            className="answer-content"
-                            html={answer.content_html} />
-                        {feedback}
-                    </div>
-                </div>
-            );
-        } else {
-            body = (
-                <>
-                    {correctIncorrectIcon}
-                    {selectedCount}
-                    {radioBox}
-                    <label
-                        onKeyPress={this.onKeyPress}
-                        htmlFor={`${qid}-option-${iter}`}
-                        className="answer-label">
-                        <span className="answer-letter-wrapper">
-                            <button
-                                onClick={onChange}
-                                aria-label={ariaLabel}
-                                className="answer-letter"
-                                disabled={disabled || isIncorrect}
-                                data-test-id={`answer-choice-${ALPHABET[iter]}`}
-                            >
-                                {ALPHABET[iter]}
-                            </button>
-                        </span>
-                        <div className="answer-answer">
-                            <ArbitraryHtmlAndMath
-                                {...htmlAndMathProps}
-                                className="answer-content"
-                                html={answer.content_html} />
-                            {feedback}
-                        </div>
-                    </label>
-                </>
-            );
-        }
-
         return (
-            <div className="openstax-answer">
-                <section role="region" className={classes}>
-                    {body}
-                </section>
-            </div>
-        );
+            <OSAnswer
+                type={type}
+                iter={iter}
+                answer={answer}
+                chosenAnswer={chosenAnswer}
+                onChangeAnswer={this.onChange}
+                disabled={disabled}
+                onKeyPress={this.onKeyPress}
+                qid={qid}
+                correctIncorrectIcon={<Icon type="check" color="green" />}
+                selectedCount={selectedCount}
+                feedback={feedback}
```

Comment:
this prop is also not used anymore - maybe this pr just doesn't have the latest changes?

Reply:
Yeah I forgot to update this with the newest changes, I'll pull those in and see where these props are used.